tensorflow/lite/delegates/gpu/gl/portable_egl.h

```diff
@@ -19,4 +19,12 @@ limitations under the License.
 #include <EGL/egl.h>
 #include <EGL/eglext.h>
 
+// Workaround for the awful decisions made in X11
```

Comment:
This is a wrong workaround.  What you want is

`--copt -DMESA_EGL_NO_X11_HEADERS --copt -DEGL_NO_X11`

and if this is the only thing that you really need, I wonder whether a global define is really necessary...

Reply:
That is exactly what I've thought, but my current `EGL/eglplatform.h` does not honor `EGL_NO_X11` (my `eglplatform.h` comes from `libegl1-mesa-dev`, version `1:20.0~git200108124800.ad4fb7e~b~padoka0` - I assume older mesa will not honor `EGL_NO_X11`)

Reply:
For the record, the preprocessor check that guards the `<X11/Xlib.h>` and `<X11/Xutil.h>` includes on my system (which is Ubuntu 18.04 + Padoka PPA) is
```C
#elif defined(__unix__) || defined(USE_X11)
```
(so even having `__unix__` defined would lead to X11 headers being included)

Reply:
Sorry, turns out the problem is on my side due to using unofficial mesa build (that did not have either `EGL_NO_X11` or `MESA_EGL_NO_X11_HEADERS` checks in place). I've reverted the `portable_egl.h` and `portable_gl31.h` to their previous versions.